tests/test_client.py

```diff
@@ -0,0 +1,38 @@
+from python_on_whales import docker
+from utils import set_netem, Cluster, Client, Server
+
+
+# Always use test_ prefix. Otherwise `pytest` will not notice test method.
+# `compose_up` prefix is an fixture defined in `conftest.py`.
+def test_client(compose_up):
```

Comment:
@staszek-krotki Here is a sample test showing usage of some `utils` and useful Python lang features.
It covers `ping` and `sessions`.
I will add `transfer` and `find_node` when working on another example/proper test